scripts/check_codebase_language.py

```
@@ -0,0 +1,177 @@
+
+import sys
+from pathlib import Path
+
+try:
+    import language_tool_python  # type: ignore
+except ImportError:
+    print("LanguageTool not installed. Run: pip install language_tool_python")
+    sys.exit(1)
+
+
+def extract_comments_from_python(file_path: str) -> list[tuple[str, int]]:
```

Ruff (0.12.2)

`extract_comments_from_python` is too complex (13 > 10)

(C901)